.github/workflows/ci-vulnerability-scans.yml

```diff
@@ -33,8 +33,16 @@ jobs:
         uses: hadolint/hadolint-action@v3.1.0
         with:
           dockerfile: app/Dockerfile
-          format: json
+          format: tty
           failure-threshold: warning
+          output-file: hadolint-results.txt
+
+      - name: Save output to workflow summary
+        run: |
+          echo "# Hadolint Results" >> $GITHUB_STEP_SUMMARY
+          if [ -f hadolint-results.txt ]; then
```

Comment:
is this if statement necessary?

Reply:
I had it from the filtering so that it wouldn't print the header of the table with no rows, but since we aren't using it now, it isn't necessary